### PR TITLE
[DOCFIX] Update aws emr doc

### PR DESCRIPTION
**Pull request by Xenorith**
add link to aws cli cmd for emr create-clusters
describe alternative method to reference json configuration file

### CODE REVIEW

File: docs/en/cloud/AWS-EMR.md [hunk 1]
```diff
@@ -71,8 +72,9 @@ A good instance type to start off with is `r4.4xlarge`.
     - You can also specify additional Alluxio properties as a delimited list of key-value pairs in the format `key=value`.
     For example, `alluxio.user.file.writetype.default=CACHE_THROUGH` instructs Alluxio to write files synchronously to the underlying storage system.
     See more about [write type options]({{ '/en/overview/Architecture.html#data-flow-write' | relativize_url }}).
-- `configurations`: The path to the configuration json file, also hosted in a publicly readable S3 bucket: `s3://alluxio-public/emr/{{site.ALLUXIO_VERSION_STRING}}/alluxio-emr.json`
-- `ec2-attributes`: EC2 settings to provide, most notably the name of the key pair to use to connect to the cluster
+- `configurations`: The path to the configuration json file, also hosted in a publicly readable S3 bucket: `https://s3.amazonaws.com/alluxio-public/emr/{{site.ALLUXIO_VERSION_STRING}}/alluxio-emr.json`
```
Comment (LuQQiu) [inline docs/en/cloud/AWS-EMR.md:75]: they can provide a http link or a s3 bucket path?
or they can only provide HTTP link?

**Review by LuQQiu — COMMENTED** [review]
Thanks for updating the doc, left one minor question, thanks

File: docs/en/cloud/AWS-EMR.md (hunk 1 above)
Comment (Xenorith) [reply docs/en/cloud/AWS-EMR.md:75]: according to the cli docs, this should be either a https url or local file. it cannot be a s3 url
```
--configurations (string)

Specifies a JSON file that contains configuration classifications, which you can use to customize applications that Amazon EMR installs when cluster instances launch. Applies only to Amazon EMR 4.0 and later. The file referenced can either be stored locally (for example, --configurations file://configurations.json ) or stored in Amazon S3 (for example, --configurations https://s3.amazonaws.com/myBucket/configurations.json ). Each classification usually corresponds to the xml configuration file for an application, such as yarn-site for YARN.
```

so this command never worked as far as i can tell

Comment (LuQQiu) [reply docs/en/cloud/AWS-EMR.md:75]: Oh dannnn,, thanks for figuring out

**Review by LuQQiu — APPROVED** [review]
Thanks for finding out the error and fix the docs!

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.5 please

**Pull request merged**